docs/en/observability/profiling-envs.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[profiling-envs]]
+= Environment variables to configure the Universal Profiling Agent
+
+The Universal Profiling Agent can be configured with environment variables.
+
+WARNING: Environment variables do have precedence over command line arguments to the Universal Profiling Agent.
```

Comment:
This is incorrect, CLI arguments have the highest [priority](https://github.com/peterbourgon/ff?tab=readme-ov-file#parse-priority).

```suggestion
```

Reply:
sorry, I have mixed that up. The warning should be vice versa and mention that environment variables are overwritten by cli arguments.